scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -584,6 +628,58 @@ public static void PublishSharedFramework(BuildTargetContext c, string outputDir
             File.WriteAllText(Path.Combine(SharedFrameworkNameAndVersionRoot, ".version"), content);
         }
 
+        private static byte[] UpdateNuGetVersion(BuildTargetContext c, string projectJsonPath, string version)
```

Comment:
Yes, I'll include that in a separate PR, showing the changes to remove this bootstrap stuff.